mava/systems/tf/mappo/training.py

```diff
@@ -483,28 +486,41 @@ def _backward_pass(self) -> None:
         """Trainer backward pass updating network parameters"""
 
         # Calculate the gradients and update the networks
-        total_loss = self.total_losses
+        policy_losses = self.policy_losses
+        critic_losses = self.critic_losses
         tape = self.tape
 
         for agent in self._agents:
             # Get agent_key.
             agent_key = self._agent_net_keys[agent]
 
-            # Get trainable variables.
-            variables = (
-                self._policy_networks[agent_key].trainable_variables
-                + self._critic_networks[agent_key].trainable_variables
+            policy_variables = self._policy_networks[agent_key].trainable_variables
```

Comment:
For DDPG we still update the obs network with both gradients. Shouldn't this PR include the fix for that as well?

Reply:
Moved to separate pr.